packages/openactive-broker-microservice/README.md

```diff
@@ -294,3 +311,548 @@ Clients credentials can be defined as follows:
+
+If an Opportunity matching the Opportunity Criteria, Seller and Opportunity Type from the request body is found, it will be returned. Otherwise, the response will be a 404.
+
+Any Opportunity returned from this endpoint will be [locked](#opportunity-locks) and so will not be available for subsequent calls until locks are released.
+
+### `DELETE /test-interface/datasets/:testDatasetIdentifier`
+
+**Request params**:
+
+* `testDatasetIdentifier`: The identifier of the [Test Dataset](https://openactive.io/test-interface/#datasets-endpoints)
+
+**Response type**: 🕳️ Empty
+
+This endpoint mirrors the [Test Interface](https://openactive.io/test-interface/)'s [`DELETE /test-interface/datasets/:testDatasetIdentifier`](https://openactive.io/test-interface/#delete-test-interfacedatasetstestdatasetidentifier) endpoint exactly. But, instead of deleting Opportunities, it releases all [Opportunity Locks](#opportunity-locks) created for the specified `testDatasetIdentifier`.
+
+In this way, it can be used by [Integration Tests](../openactive-integration-tests/) when it is running in [Random mode](../openactive-integration-tests/README.md#userandomopportunities) to get ready for a new test run.
+
+### `POST /assert-unmatched-criteria`
+
+**Request body**: The shape of [Child Opportunity](#orphans) to check. This data has the same specification as the request body to [Test Interface](https://openactive.io/test-interface/)'s [`POST /test-interface/datasets/:testDatasetIdentifier/opportunities`](https://openactive.io/test-interface/#post-test-interfacedatasetstestdatasetidentifieropportunities) endpoint.
+
+**Response type**: 🤖 JSON
+
+Assert that an Opportunity matching the Opportunity Criteria, Seller and Opportunity Type from the request body is **not** found in the [Booking System](#booking-system-under-test)'s data. If such an Opportunity is found, the response will be a `404`. Otherwise, the response will be a `204`.
+
+This is used by some [Non-Implemented tests](../openactive-integration-tests/README.md#structure) to ensure that a test feature MUST be implemented if a certain criteria of Opportunity is found in the Booking System's data. e.g. the [`cancellation-window` feature](../openactive-integration-tests/test/features/cancellation/cancellation-window) has a non-implemented test that asserts that there are no Opportunities with cancellation windows defined.
+
+## Browser Automation for Auth Endpoints
+
+Broker Microservice also exposes endpoints created from the `setupBrowserAutomationRoutes(..)` function from [openactive-openid-test-client](../openactive-openid-test-client/).
+These endpoints are used by [Integration Tests](../openactive-integration-tests/) for tests which check the [OpenID Connect](https://openid.net/developers/how-connect-works/) authentication flow.
+
+## Concepts
+
+### Auth
```

Comment:
```suggestion
### Authentication
```

Reply:
My understanding is that Broker also manages authorization as well as authentication e.g. it maintains the booking partners' access tokens.

Reply:
Great point - something I've noticed is an issue in the booking spec too (and have raised https://github.com/openactive/open-booking-api/issues/240). Perhaps "Authentication and Authorisation" could be a good solution here too?

Reply:
However reflecting on this - in this specific case, is the configuration here more focussed on Authentication than Authorisation?